core/src/main/java/org/apache/accumulo/core/fate/AbstractFateStore.java

```diff
@@ -502,4 +520,14 @@ protected Serializable deserializeTxInfo(TxInfo txInfo, byte[] data) {
       throw new IllegalStateException("Bad node data " + txInfo);
     }
   }
+
+  /**
+   * TODO 4131 this is a temporary method used to create a dummy lock when using a FateStore outside
```

Comment:
Yes, that sounds like a great change. And yes you are correct about some of the fate admin utils requiring the Manager to be down, so that would be helpful for those as well.